src/main/java/pages/checkoutConfirmationPage/CheckoutConfirmationPage.java

```diff
@@ -0,0 +1,59 @@
+package pages.checkoutConfirmationPage;
+
+import org.openqa.selenium.By;
+import pages.BasePage;
+
+import java.math.BigDecimal;
+
+public class CheckoutConfirmationPage {
+
+    protected BasePage basePage;
+
+    public By returnPolicyLocator = By.xpath("//b[normalize-space()='Return Policy']");
+    public By closeReturnPolicyCrossLocator = By.xpath("//*[@id='returnPolicyModal']/div/div/div[1]/button");
+    public By closeReturnPolicyButtonLocator = By.xpath("//*[@id='returnPolicyModal']/div/div/div[3]/button");
+    public final String totalsTable = "//div[contains(@class, 'confirm_total')]//table";
+    public By subTotalLocator = By.xpath(totalsTable + "/tbody/tr[1]/td[2]/span");
+    public By flatShippingRateLocator = By.xpath(totalsTable + "/tbody/tr[2]/td[2]/span");
+    public By totalLocator = By.xpath(totalsTable + "/tbody/tr[3]/td[2]/span");
+    public By backButtonLocator = By.id("back");
+    public By confirmOrderButtonLocator = By.id("checkout_btn");
+
+    public CheckoutConfirmationPage() {
+        this.basePage = new BasePage();
+    }
+
+    public void clickOnReturnPolicy() {
+        basePage.clickOnElement(returnPolicyLocator);
+    }
+
+    public void clickOnCloseReturnPolicyByCross() {
+        basePage.clickOnElement(closeReturnPolicyCrossLocator);
+    }
+
+    public void clickOnCloseReturnPolicy() {
+        basePage.clickOnElement(closeReturnPolicyButtonLocator);
+    }
+
+    public BigDecimal getSubTotalValue() {
+        return new BigDecimal(basePage.getTextFromElement(subTotalLocator).trim());
+    }
+
+    public BigDecimal getFlatShippingRateValue() {
```

Comment:
Why BigDecimal?

Reply:
I had void before change. I think this is a good idea because the amounts are numbers and are floating point. What do you think?